state/app/PreferenceManager.kt

```diff
@@ -16,23 +16,25 @@
  *
  */
 
-package com.vaticle.typedb.studio.state.common.util
+package com.vaticle.typedb.studio.state.app
 
 import java.nio.file.Path
 import kotlin.io.path.name
 
-// TODO: make this file configurable through a Preference Dialog
-class PreferenceManager {
+class PreferenceManager(appData: DataManager) {
```

Comment:
Currently `PreferenceManager` seems like a thin abstraction, but in the future preferences may become more complex and having an interface to provide data about the preferences will be essential.

Reply:
Implemented. See above comment, much more enthusiastic about this.